### ISSUE TITLE
Phases of development

### ISSUE THREAD

**Issue by jwentwistle** [mitar/council-app#42]
I think you should add phases to each of the decision making process. In particular, to represent an efficient version of what happens at city councils. The other decision making tools allow voting on decisions and discussion about it, but they don't build into further collaboration on those decisions. 

Such that, you can do the following: 

-Vote on an idea (initial)
-Vote on components of the idea to be implemented by specialized committees (implementation)
-Vote on the legal wording of the implementation (legalizing)
-Allowing the final legal wording to be budgeted in terms of costs and benefits
-Vote by everybody on final version

**Comment by mitar** [comment]
You are right. Currently, I am trying somehow to convey this through three panels (process goes from left to right) and that users can upvote content on first two panels, and then vote with final vote in the last panel.

**Comment by mitar** [comment]
So one approach to address your proposal would be just to add upvoting on the discussions themselves.

> Vote on the legal wording of the implementation

About this, I have written some ideas about this [in this blog post](http://mitar.tnode.com/post/73930434538/massive-online-collaborative-decision-making). Not sure when this would be implemented, though.

**Comment by mitar** [comment]
In a longer term, I would like this app to have customizable panels, so for a particular discussion you could create your own series of panels. Then this ticket, based on my reading (feel free to correct me) would be translated more to create specialized panels for each of those steps your are describing. So that people can then do those steps if they need it.

**Comment by jwentwistle** [comment]
Yeah, I'm not sure as to the best way to implement the idea. I was just suggesting it and thought that if city councils used such a software it may improve their process.

**Comment by mitar** [comment]
Related: #60
